packages/expo-prevent-screen-capture/ios/EXPreventScreenCapture/EXPreventScreenCaptureModule.m

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import <EXPreventScreenCapture/EXPreventScreenCaptureModule.h>
+
+@interface EXPreventScreenCaptureModule ()
+
+@property (nonatomic, weak) UMModuleRegistry *moduleRegistry;
+@property (nonatomic, strong) UIView *blockView;
+
+@end
+
+@implementation EXPreventScreenCaptureModule
+
+UM_EXPORT_MODULE(ExpoPreventScreenCapture);
+
+- (id)init {
+    if ( self = [super init] ) {
+        self.blockView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, [[UIScreen mainScreen] bounds].size.width, [[UIScreen mainScreen] bounds].size.height)];
```

Comment:
Forgive me if I'm wrong, but does it still work ok for different screen orientations?

Reply:
it does now! thanks for catching this, changed to use `[[UIScreen mainScreen] bounds].size.height` for both bounds